docs/IWYUPragmas.md

```diff
@@ -1,23 +1,21 @@
-## IWYU pragmas ##
+# IWYU pragmas #
 
 IWYU pragmas are used to give IWYU information that isn't obvious from the source code, such as how different files relate to each other and which includes to never remove or include.
 
-All pragmas start with `// IWYU pragma: ` or `/* IWYU pragma: `. They are case-sensitive and spaces are significant.
```

Comment:
The trailing spaces in the pragma examples were intended, that's really the pattern that pragmas start with.

Reply:
Okay. Those trailing spaces are omitted in the generated HTML when using backticks for inline HTML. I've thus changed to use <code> tags in these instances.

Reply:
Hm, that screws up the Markdown for the sake of one renderer. I'd like for the Markdown source to be copy/paste-friendly and readable. Let's put this back the way it was.